niworkflows/interfaces/masks.py

```diff
@@ -113,7 +113,7 @@ def _run_interface(self, runtime):
 
         mask_nii = compute_epi_mask(padded_nii, exclude_zeros=True)
 
-        mask_data = mask_nii.get_data()
+        mask_data = mask_nii.get_fdata()
```

Comment:
Presumably uint8s are better than floats here...

```suggestion
        mask_data = np.asanyarray(mask_nii.dataobj).astype(np.uint8)
```